### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HarshalNeelkamal**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

`./hack/update-vendor.sh` is step 7 combines step 7 and 8 making step 8 redundant. This CL cleans by removing step 8.

#### Which issue(s) this PR fixes:

N/A

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
N/A
```

### CODE REVIEW

File: staging/README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ import (
   ./hack/update-vendor.sh
 ```
 
-8. Run [`./hack/update-go-workspace.sh`](/hack/update-go-workspace.sh) to add
+8. Run [`./hack/update-vendor.sh`](/hack/update-vendor.sh) to add
```
Comment (liggitt) [inline staging/README.md:90]: This script is already referenced in step 7, combine 7 and 8

Comment (HarshalNeelkamal) [reply staging/README.md]: Oops! didn't remember that I already ran it by the time I got to updating. 

Have removed step 8.

**Pull request merged**